src/docs/upgrading.dox

```diff
@@ -1,5 +1,20 @@
 /*! @page upgrading Upgrading WiredTiger applications
+@section version_310 Upgrading to Version 3.1.0
+<dl>
+
+<dt>WiredTiger on-disk log file format change</dt>
+<dd>
+The WiredTiger on-disk file format for write-ahead log files has changed
+as the log file version number was incremented.
+</dd>
 
+<dt>::wiredtiger_open compatibility configuration changes</dt>
+<dd>
+The compatibility setting now takes additional options that can define
+the minimum or maximum required version of existing data files.
```

Comment:
Should we provide an example to show how to set the 'require_min' & 'require_max' options here, or it's been covered in other documentations so that we can simply add a reference link?